File: week6-python/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

x = 0

while x < 1 or x > 8:
    x = get_int("Height: ")

for i in range(x):
    print(" " * (x - 1 - i), end="")
    print("#" * (i + 1))

File: week1-c/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void print_grid(int height);

int main(void)
{
    // get height
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    print_grid(n);

    return 0;
}

void print_grid(int height)
{
    for (int i = 0; i < height; i++)
    {
        // print space
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }

        // print block
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        // to next line
        printf("\n");
    }
}